packages/teams-js/src/public/dialog.ts

```diff
@@ -17,21 +17,30 @@ import { runtime } from './runtime';
  *
```

Comment:
Is the comment on line 16 true? I think some of these functions work in other FrameContexts (like task for example)

Reply:
does not seem true, thank you for catching. 

dialog.open works in 3 frame contexts,  but submit and resize work from 4 fram contexts. `sendMessageToParentFromDialog` works only on task. 

What should we update the remark to?

Reply:
does not seem true, thank you for catching. 

dialog.open works in 3 frame contexts,  but submit and resize work from 4 fram contexts. `sendMessageToParentFromDialog` works only on task. 

What should we update the remark to?

Reply:
I would probably remove this remark and if necessary add remarks to individual functions. For example, it might be worth calling out that you can't call dialog.open from a dialog